src/components/Dialogs/CreateFormDialog/CreateFormTrigger/tests/CreateFormTrigger.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { vi } from 'vitest';
+
+import { render, screen, cleanup } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import CreateProductPortal from '@/components/Dialogs/CreateProductDialog/CreateProductPortal';
+import CreateCategoryPortal from '@/components/Dialogs/CreateCategoryDialog/CreateCategoryPortal';
+
+import CreateFormTrigger from '..';
+
+vi.mock(`@/components/Dialogs/CreateProductDialog/CreateProductPortal`);
+vi.mock(`@/components/Dialogs/CreateCategoryDialog/CreateCategoryPortal`);
+
+vi.mock(`@clerk/nextjs`, () => ({
+  __esModule: true,
+  useUser: () => ({ user: { id: `1` } }),
+}));
+
+describe(`CreateFormTrigger`, () => {
+  afterAll(() => {
+    vi.clearAllMocks();
+  });
+
+  afterEach(() => {
+    cleanup();
+    vi.resetAllMocks();
+  });
+
+  it(`When CreateFormTrigger is called with formType prop equal to product-form, should be rendered the CreateProductPortal component`, async () => {
```

Comment:
```suggestion
  it(`When CreateFormTrigger is called with formType prop equal to product-form and title prop equal to 'Produtos', should be rendered the CreateProductPortal component`, async () => {
```

Reply:
This is unnecessary because I only test elements with conditional rendering

Reply:
but in this test you use title 'Produtos' and the other test you use title 'Categorias', if you don't title it that way, both tests will have the same title.

Reply:
But title is passed by params and not is conditionally, and the test case not have an expect to test if the title is product or category, why it’s not necessary

Reply:
The title can be xablau, not matter what title you pass, the title ever be what’s passed by params

Reply:
yes, but you don't think that's strange two test with the same name? isn't better to be only one test, to test both?

Reply:
But I already fix it @ebertonArouxa